scripts/create-seed-fixture-heartwood

```diff
@@ -27,25 +27,68 @@ mkdir -p $TEST_REPO_PATH
 
 tar -xf $TEST_REPO_ARCHIVE -C $TEST_REPO_PATH
 
-RAD_HOME=$PALM_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad auth
+### ALICE ###
+
+export RAD_HOME=$ALICE_RAD_HOME
+export RAD_SEED=ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
 
-GIT_AUTHOR_NAME="Alice Liddell"
-GIT_AUTHOR_EMAIL="alice@radicle.xyz"
-GIT_COMMITTER_NAME="Alice Liddell"
-GIT_COMMITTER_EMAIL="alice@radicle.xyz"
+export GIT_AUTHOR_NAME="Alice Liddell"
+export GIT_AUTHOR_EMAIL="alice@radicle.xyz"
+export GIT_COMMITTER_NAME="Alice Liddell"
+export GIT_COMMITTER_EMAIL="alice@radicle.xyz"
 
-RAD_SEED=ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad auth
+eval $(ssh-agent)
+rad auth
+rad auth # first `rad auth` won't add key to ssh-agent
+radicle-node --listen 0.0.0.0:3444 &
 
 cd $ALICE_CHECKOUT
 
 git clone $TEST_REPO_PATH
 cd $TEST_REPO_NAME
+git checkout main
+
+rad init --name "source-browsing" \
+	--description "Git repository for source browsing tests" \
+	--default-branch "main" \
+	--no-confirm
+
 git checkout feature/branch
+git push rad
+
 git checkout orphaned-branch
-git checkout main
+git push rad
+
+### BOB ###
 
-RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad init --name "source-browsing" --description "Git repository for source browsing tests" --default-branch "main" --no-confirm
-PROJECT_ID=$(RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad .)
+export GIT_AUTHOR_NAME="Bob Belcher"
+export GIT_AUTHOR_EMAIL="bob@radicle.xyz"
+export GIT_COMMITTER_NAME="Bob Belcher"
+export GIT_COMMITTER_EMAIL="bob@radicle.xyz"
+export GIT_COMMITTER_DATE="Mon Nov 21 14:00 2022 +0100"
```

Comment:
Bob's commit doesn't match with non-heartwood version and the culprit I supposed is `GIT_COMMITTER_DATE`.

Reply:
Is this necessary when we already have an `if` in the specs?

Reply:
Try Dec 21.